components/card/src/vwc-card.ts

```diff
@@ -63,37 +63,42 @@ export class VWCCard extends LitElement {
 	}
 
 	protected override render(): unknown {
-		const actionsClassMap = {
-			'no-content': !(this.shouldShowActionsSlot)
+		const footerClassMap = {
+			'no-content': !(this.shouldShowFooterSlot)
 		};
 		return html`
 			<div class="vwc-card">
 				<div class="vwc-card-media">
 					<slot name="media"></slot>
 				</div>
-				<div class="vwc-card-info">
-					${this.renderHeader()}
-					<div class="vwc-card-supportText">
+				<div class="vwc-card-content">
+					<slot name="content">
+						${this.renderHeader()}
+						<div class="vwc-card-supportText">
 							${this.supportingText ? this.supportingText : ''}
```

Comment:
Instead of empty string here, can use the `nothing` directive

Reply:
changed it